temporal-sdk/src/main/java/io/temporal/client/schedules/Schedule.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright (C) 2022 Temporal Technologies, Inc. All Rights Reserved.
+ *
+ * Copyright (C) 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this material except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.temporal.client.schedules;
+
+import java.util.Objects;
+import javax.annotation.Nonnull;
+
+/** A schedule for periodically running an action. */
+public final class Schedule {
+
+  public static Schedule.Builder newBuilder(
+      @Nonnull ScheduleAction action, @Nonnull ScheduleSpec spec) {
+    return new Schedule.Builder(action, spec);
+  }
+
+  public static Schedule.Builder newBuilder(Schedule options) {
+    return new Schedule.Builder(options);
+  }
+
+  private Schedule(
+      ScheduleAction action, SchedulePolicy policy, ScheduleState state, ScheduleSpec spec) {
+    this.action = action;
+    this.policy = policy;
+    this.state = state;
+    this.spec = spec;
+  }
+
+  /**
+   * Gets the policy for the schedule.
+   *
+   * @return policy of the schedule
```

Comment:
`@return` is required for  any method that does not return void, according to oracle at least

Reply:
To confirm, you're saying "`@return` is not required unless you document the method then it becomes required?" I don't think that's valid even if that's what they suggest for javadoc. I can show a ton of cases in our current SDK that does not use `@return`. Oracle may suggest things, but does not mean this project adopted them.

Reply:
"@return is required on any non void you document" I agree there are lots of cases in the SDK that do not follow, that doesn't make it not required. We just disable all javadoc linting.

Reply:
I think it makes it not required, especially since we disabled requiring it. I do not think we should listen to Oracle here if that's what they say. I believe the same thing about `@param` for setters/obvious, `@throws`, etc.

Also, if you are under the impression that `@return` is truly required then javadoc itself is required on everything that has a non-void return, which is not the case in this PR.